### PR TITLE
WT-10602 Check alternative path delimiter on Windows in tests

### PR DESCRIPTION
**Pull request by pmacko86**

### CODE REVIEW

**Review by korteland — COMMENTED** [review]
Thanks for the quick investigation and turnaround on this ticket!

File: test/utility/misc.c [hunk 1]
```diff
@@ -81,10 +81,20 @@ testutil_die(int e, const char *fmt, ...)
...
-        progname = argv[0];
+    char *str;
+#ifdef _WIN32
+    /*
+     * On some Windows environments, such as CYGWIN, argv[0] can use '/' as a path delimiter instead
+     * of '\\', so check both just in case.
+     */
+    if ((str = strrchr(argv[0], '/')) != NULL)
+        progname = str + 1;
     else
```
[the remark this answers is not in the document]
Comment (pmacko86) [reply test/utility/misc.c]: I see how this code structure can be confusing, so I rewrote it without "else".

File: test/utility/misc.c [hunk 2]
```diff
@@ -81,10 +81,20 @@ testutil_die(int e, const char *fmt, ...)
 const char *
 testutil_set_progname(char *const *argv)
 {
-    if ((progname = strrchr(argv[0], DIR_DELIM)) == NULL)
-        progname = argv[0];
+    char *str;
+#ifdef _WIN32
+    /*
+     * On some Windows environments, such as CYGWIN, argv[0] can use '/' as a path delimiter instead
```
[the remark this answers is not in the document]
Comment (pmacko86) [reply test/utility/misc.c]: Fixed.

**Review by korteland — APPROVED** [review]
Thanks Peter! LGTM.

**Pull request merged**